### PR TITLE
Markdown file

### PR DESCRIPTION
**Pull request by IngridGdesigns**
corrected typo in name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +112,7 @@ The input file is a JSON file in the format:
 - Example complete url: "https://api.github.com/repos/spring-projects/spring-framework": 19805,
```
Comment (kevindigo) [inline README.md:112]: This line and the one after should have backticks around the content that would actually appear in a config file.

File: src/Utilities/generateConfluenceMarkup.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
 import { DateTime } from 'luxon';
 import { Issue } from '../mariner';
+import { calculateAgeInWholeDays, removeBracesAndBrackets } from '../Utilities/outputHelpers';
 
 export function generateConfluenceMarkup(
     issuesByDependency: Map<string, Issue[]>,
-    maxIssuesAge = 30
+    maxIssuesAge = 430
```
Comment (kevindigo) [inline src/Utilities/generateConfluenceMarkup.ts:7]: Was it intentional to change from 30 days to 430?

File: src/__tests__/generateConfluenceMarkup.test.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ const fakeIssues: Issue[] = [
 
 const singleIssue: Issue[] = [
     {
-        title: 'ToC: links to markdown headings',
+        title: 'ToC: links to markup headings',
```
Comment (kevindigo) [inline src/__tests__/generateConfluenceMarkup.test.ts:30]: In this case, the title was whatever the actual sample issue had, so it doesn't matter to us whether it was markup, markdown, or something else. To avoid confusion, let's delete the word "markup".

File: src/mariner/index.ts [hunk 4]
```diff
@@ -2,5 +2,6 @@ export { DependencyDetailsRetriever } from '../dependency-details-retriever';
 export { readConfigFile, Config } from '../config';
 export { Issue, IssueFinder } from '../issueFinder';
 export { Logger, getLogger, setLogger } from '../tab-level-logger';
-export { generateConfluenceMarkup as generateConfluenceMarkdown } from '../Utilities/generateConfluenceMarkup';
+export { generateConfluenceMarkup } from '../Utilities/generateConfluenceMarkup';
```
Comment (kevindigo) [inline src/mariner/index.ts:5]: is it intentional that this is a breaking change, since it renames the exported confluence output function? I am inclined to think that the breaking change should probably be a separate PR.

**Review by kevindigo — CHANGES_REQUESTED** [review]

File: src/mariner/index.ts (hunk 4 above)
Comment (IngridGdesigns) [reply src/mariner/index.ts:5]: yes it's intentional, since we are adding in another file, [generateGitHubMarkdown](https://github.com/indeedeng/Mariner/pull/111/files#diff-c7afdb73d2ada6c84b773c723c7d964fe259594aba2d6118c9272dc6ee993f30) and we also have [generateGitHubMarkup](https://github.com/indeedeng/Mariner/pull/111/files#diff-def8a2a22dd145c0da7302d9204fe4d8c9b233248707edaad534a845af6e0d33) it was renamed to markup instead of markdown

File: README.md (hunk 1 above)
Comment (IngridGdesigns) [reply README.md]: added [backticks](https://github.com/indeedeng/Mariner/pull/111/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

**Review by kevindigo — APPROVED** [review]
